src/ibims/bo/marktlokation_erweitert.py

```diff
@@ -22,3 +22,11 @@ class MarktlokationErweitert(Marktlokation):
             Z15: WorkPricePerformancePrice (Arbeitspreis/Leistungspreis)
     """
     community_id: str
+    prognose_grundlage: Literal["ZA6", "ZC0", None]
+    """
+    forecast type of a market location, ZA6: "Prognose auf Basis von Werten", ZC0: "Prognose auf Basis von Profilen"
+    """
+    prognose_grundlage_detail: Literal["E02", "E14", "Z36", None]
+    """
+    forecast detail of a market location, E02: SLP/SEP, E14: TLP/TEP, Z36: TEP mit Referenzmessung
+    """
```

Comment:
das sind jetzt halt die 1:1 die EDIFACT-Werte. Können wir nicht eine Bo4E-Entsprechung finden oder ein menschenlesbares Enum aufmachen?

Reply:
Es gibt soweit ich das sehe keine offizielen BO4E entsprechungen (die Prognosegrundlage gibt es aber in der go/c# Version) und ich mach deshalb enums auf.

Aber was ist eigentlich mit den ganzen anderen Feldern, in denen wahrscheinlich irgendwelche Edifact-Codes reinlaufen und welche nur als Strings typisiert sind? Wollen wir das noch ändern? Wir haben einige davon, z. B. im Verbrauch https://github.com/Hochfrequenz/intermediate-bo4e-migration-models/blob/3131d75da2877f392a6db129fa71711e8bd9b772/src/ibims/com/verbrauch.py#L45.

Reply:
ja, den ablesegrund sollten wir langfristig besser machen. Denn ein string ist keine gute Typisierung ;) Das Literal ist ein Schritt in die richtige Richtung aber ich würde lieber Enums aus C#/Go kopieren als die EDIFACT-Werte ins Zwischenmodell zu packen.

Es gibt auch Gründe, das anders zu sehen, die kann Matthias nennen. Nämlich dann, wenn letztlich EDIFACT der einzig sinnvolle Standard ist und weitere ENUMs, die nur EDIFACT verpacken lediglich sowas wie Boilerplate wären.